config/initializers/deprecations.rb

```diff
@@ -1 +1,29 @@
 ActiveSupport::Deprecation.behavior = :silence if Rails.env.production?
+
+Foreman::Application.configure do |app|
+  config.after_initialize do
+    unless Foreman.in_rake?
+      if ActiveRecord::Base.connection.adapter_name.downcase.starts_with?('mysql')
+        blueprint = NotificationBlueprint.find_by_name('feature_deprecation')
+        return unless blueprint
```

Comment:
there is no function - you are in a block, you can't return, you should use `break` instead:
```suggestion
        break unless blueprint
```